Core.DomainModel/Organization/OrganizationType.cs

```diff
@@ -3,6 +3,12 @@
 
 namespace Core.DomainModel.Organization
 {
+    /// <summary>
+    /// 1 - Kommune
+    /// 2 - Community of interest
+    /// 3 - Company
+    /// 4 - Other public authority
+    /// </summary>
```

Comment:
I added these translation to remember which value is what. Are they accurate?

Reply:
**V2 has the answer**
Presentation.Web\Models\API\V2\Types\Organization\OrganizationType.cs

![image](https://user-images.githubusercontent.com/13116889/224268084-25086534-8227-47cd-b639-3f5f3ab96fb5.png)